Gopkg.toml

```diff
@@ -16,7 +16,7 @@
 
 [[constraint]]
   name = "github.com/opencontainers/runc"
-  revision = "0351df1c5a66838d0c392b4ac4cf9450de844e2d"
+  revision = "c35c2c9cec6ee503ef31edbaddac9617247ec328"
```

Comment:
I know this is dnm for now, but when that label is removed, could you add the changes introduced by these re-vendors in one of the commits on this PR?